components/core/Logo.vue

```diff
@@ -28,13 +27,16 @@ export default {
       required: true
     }
   },
-  data() {
-    const storeView = currentStoreView();
-    return {
-      defaultTitle: storeView.seo.defaultTitle
-        ? storeView.seo.defaultTitle
-        : config.seo.defaultTitle
-    };
+  computed: {
+    defaultTitle() {
+      const storeView = currentStoreView();
```

Comment:
Moim zdaniem wywołanie funkcji currentStoreView najlepiej przenieść poza export i tam przypisać aktualny storeView do jakiejś stałej. W tym przypadku, jak to jest w computed, to ta funkcja wywoływana jest mnóstwo razy (za każdym razem, jak computed jest przeliczane).

Reply:
done